src/sharedHooks/useLocalObservations.js

```diff
@@ -1,14 +1,22 @@
 // @flow
 
+import { useIsFocused } from "@react-navigation/native";
 import { RealmContext } from "providers/contexts";
 import {
-  useEffect, useState
+  useEffect, useRef,
+  useState
 } from "react";
 import Observation from "realmModels/Observation";
 
 const { useRealm } = RealmContext;
 
 const useLocalObservations = ( ): Object => {
+  const isFocused = useIsFocused( );
+  // use a ref as a temporary store because MyObservations page doesn't unmount on blue
+  // only updating state when focused will help prevent MyObservations from
+  // blocking other components from rendering
```

Comment:
How about something like, "Use refs to maintain state without triggering re-renders of hook consumers when they have lost focus, which prevents other views from rendering when they have focus."